aperturedb/sqlc/migrations/000003_lnc-sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS sessions (
+    id INTEGER PRIMARY KEY,
+
+    passphrase_entropy BLOB NOT NULL UNIQUE,
+    remote_static_pub_key BLOB UNIQUE,
+    local_static_priv_key BLOB NOT NULL,
+    mailbox_addr TEXT NOT NULL,
+
```

Comment:
Maybe also a string label?

Reply:
Looks like the next commit assumes one exists.

Reply:
I wasn't sure about adding one but ended up adding it. 

A session is then identified by its label or its passphrase. Which is kind of weird because a passphrase is what defines the session. I added the field so it is more human friendly though.